sdk/tables/azure-data-tables/inc/azure/data/tables/enum_operators.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <type_traits>
+
+namespace Azure { namespace Data { namespace Tables {
+  /**
+   * @brief Bitwise OR operator for enum class.
+   */
+  template <class E, class = std::enable_if_t<std::is_enum<E>{}>> E operator|(E lhs, E rhs)

```

Comment:
`constexpr`?
I'm only marking the first instance, but it can be done to all operators, including the op-and-assign forms.
```suggestion
  template <class E, class = std::enable_if_t<std::is_enum<E>{}>> constexpr E operator|(E lhs, E rhs)
```